cleo_sdk/CLEO_Utils.h

```diff
@@ -206,6 +211,38 @@ namespace CLEO
         return info;
     }
 
+    struct StringParamBufferInfo
+    {
+        char* data = nullptr;
+        int size = 0;
+        BOOL needTerminator = false;
+    };
+
+    static void MemPatchJump(size_t position, void* jumpTarget)
```

Comment:
can't find where it's used or moved from

Reply:
https://github.com/cleolibrary/CLEO5/blob/master/source/Mem.h#L22